docker/centos-graal-maven-s2i/README.md

```diff
@@ -0,0 +1,30 @@
+# [Quarkus.io](http://quarkus.io) native S2I
+
+## How to build this S2I builder image
+
+### OpenShift
+
+    oc new-build https://github.com/quarkusio/quarkus.git --context-dir=docker/centos-graal-maven-s2i --name quarkus-native-s2i
+
+### Locally
+
+    docker build . -t quarkus-native-s2i
+
+## How to use this
+
+### OpenShift
+
+    oc new-app quarkus-native-s2i~https://github.com/quarkusio/quarkus-quickstarts --context-dir=getting-started-native --name=getting-started-native
```

Comment:
Should we remind the user to have "installed" with the command mentioned previously?

Reply:
I'll re-shuffle the documents sections to make more sense, see if you like it better after the next update.